ScoutSuite/providers/aws/resources/s3/service.py

```diff
@@ -0,0 +1,52 @@
+from ScoutSuite.providers.aws.facade.facade import AWSFacade
+from ScoutSuite.providers.aws.resources.regions import Regions
+from ScoutSuite.providers.aws.resources.resources import AWSResources, AWSCompositeResources
+
+from ScoutSuite.providers.utils import get_non_provider_id
+
+
+class Bucket(AWSResources):
```

Comment:
```suggestion
class Buckets(AWSResources):
```

Reply:
Dammit. This convention makes so much sense, but it's so easy to forget.